plugins/home/package.json

```diff
@@ -48,6 +49,7 @@
     "@rjsf/utils": "5.7.3",
     "@rjsf/validator-ajv8": "5.7.3",
     "@types/react": "^16.13.1 || ^17.0.0",
+    "date-fns": "^2.30.0",
```

Comment:
https://backstage.io/docs/architecture-decisions/adrs-adr010 🙏

Reply:
Thank you, I will change the implementation!

Reply:
Fixed on 9ac767c6f7264ee946fdeaccf9634988c3bef9d8